Microsoft.PowerShell.WhatsNew/relnotes/What-s-New-in-PowerShell-72.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: What's New in PowerShell 7.2
 description: New features and changes released in PowerShell 7.2
-ms.date: 10/27/2021
+ms.date: 11/16/2022
```

Comment:
Same as above

Reply:
No, that is when the file was last updated.